Content.Server/Storage/EntitySystems/SpawnItemsOnUseSystem.cs

```diff
@@ -82,7 +82,7 @@ private void OnUseInHand(EntityUid uid, SpawnItemsOnUseComponent component, UseI
 
             if (component.Sound != null)
             {
-                _audio.PlayPvs(component.Sound, uid);
+                _audio.PlayPvs(component.Sound, args.User); // sound wasn't working until i changed it to args.User (brainfood).
             }
```

Comment:
The sound is supposed to be from the uid.

Reply:
I moved it to the spawned entity. For whatever reason when set to uid it doesnt play any audio.